### PR TITLE
Rename defs to entity_models. Simplify Jinja rendering

### PR DESCRIPTION
**Pull request by gilesknap**

### CODE REVIEW

File: src/ibek/params.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
 ]
 
 
-class DefinesTypes(Enum):
+class DefineTypes(Enum):
```
Comment (GDYendell) [inline src/ibek/params.py:20]: I think this still has one too many `s`s

File: src/ibek/ioc.py [hunk 2]
```diff
@@ -54,60 +55,69 @@ class Entity(BaseSettings):
     entity_enabled: bool = Field(
         description="enable or disable this entity instance", default=True
     )
-    __definition__: EntityDefinition
+    __definition__: EntityModel
```
Comment (GDYendell) [inline src/ibek/ioc.py:58]: This should probably have a new name too

File: src/ibek/params.py (hunk 1 above)
Comment (gilesknap) [reply src/ibek/params.py:20]: agreed

File: src/ibek/ioc.py (hunk 2 above)
Comment (gilesknap) [reply src/ibek/ioc.py:58]: I'm not getting you - that is the new name we settled on

**Pull request merged**